02_cancer_type_classification/lasso_range_analysis/lasso_range_gene_cancer_type.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>Its very interesting that the stratified holdout and the test peak very close to each other.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>It was interesting to us too! I think it's pretty typical though, at least for this mutation prediction problem across genes and cancer types. That's one of the main takeaways from our paper, that cross-validation performance is generally pretty predictive of generalization performance, and model simplicity may not be as important.</p>